lib/index.js

```diff
@@ -28,9 +26,9 @@ const {checkStatus} = require('./status');
 const {'vc-verifier': cfg} = config;
 
 // must be loaded after the module config
-const loader = require('./documentLoader');
+const {documentLoader} = require('./documentLoader');
 
-const api = {loader};
+const api = {documentLoader};
```

Comment:
@mattcollier The documentLoader instance is being used in the test, for example here https://github.com/digitalbazaar/bedrock-vc-verifier/blob/support-ed25519-2020/test/mocha/helpers.js#L10 and  this was how it was implemented before, I just changed the name from loader to documentLoader, is there a better way to do this? When I tried to  require the documentLoader in the test directly from the file `documentLoader.js`, I get the error -  Cannot find module 'bedrock-https-agent' even though I do have that in the test node modules.

Reply:
Yes, if something is being exposed for testing, we normally do two things.
1) export as `_documentLoader`, the leading underscore denotes that it is not a public API, and signals to anyone using it that it may be remove/broken at any time.
2) add a comment at the point of export saying that it's exported for use in the test suite.

Reply:
Updated here 464e128